Magic/src/main/java/com/elmakers/mine/bukkit/integration/VaultController.java

```diff
@@ -102,10 +97,4 @@ public boolean depositPlayer(Player player, double amount) {
         EconomyResponse response  = economy.depositPlayer(player, amount);
         return response.transactionSuccess();
     }
-
-    @Nullable
-    public String getItemName(Material material, short data) {
-        ItemInfo info = Items.itemByType(material, data);
-        return info == null ? null : info.getName();
-    }
-}
+}
```

Comment:
Missing newline at end of file?
(`mvn checkstyle` should warn for this I think)

Reply:
Didn't notice any warning but I'll restore it anyway!